server/mongo/src/storage.ts

```diff
@@ -562,7 +562,10 @@ abstract class MongoAdapterBase implements DbAdapter {
     cursor.maxAwaitTimeMS(30000)
 
     const res = await cursor.toArray()
-    return toFindResult(res, total)
+    const result = res.map((v) => {
```

Comment:
We should do it only on client.

Reply:
Reworked to get mixin after getting query result on employee request